src/Optimizely/Event/Builder/EventBuilder.php

```diff
@@ -102,7 +103,7 @@ private function getCommonParams($config, $userId, $attributes)
             foreach ($attributes as $attributeKey => $attributeValue) {
                 $feature = [];
                 // Do not discard attribute if value is zero or false
-                if (!is_null($attributeValue)) {
+                if (Validator::isAttributeValid($attributeKey, $attributeValue)) {
```

Comment:
Comment before this line: # Omit attributes that are not supported by the log endpoint.